ticktok/ticks.py

```diff
@@ -0,0 +1,32 @@
+from threading import Thread
+from time import sleep
+
+import requests
+
+
+class Ticker:
+
+    def __init__(self):
+        self._consumers = {}
+        self._is_running = False
+
+    def register(self, url, callback):
+        self._consumers[url] = callback
+
+    def start(self):
+        if not self._is_running:
+            self._is_running = True
+            Thread(target=self.poll_ticks).start()
+
+    def poll_ticks(self):
+        while self._is_running:
+            consumers = self._consumers.copy()
+            for u, c in consumers.items():
+                response = requests.get(u)
+                if response.json():
+                    Thread(target=c).start()
+                sleep(0.01)
+
+    def unregister_all(self):
+        self._is_running = False
```

Comment:
is this bookkeeping safe? is there a way another run with a different tread will override it?

Reply:
currently it's not thread safe, that's ok for now